lib/mfem/Utilities.cpp

```diff
@@ -62,4 +62,43 @@ void ComputeCtAB_vec(const HypreParMatrix& A,
     C.transposeMult(AB_carom, CtAB_vec);
 }
 
+void verify_within_portion(const mfem::Vector &bb_min,
+                           const mfem::Vector &bb_max,
+                           const mfem::Vector &t, const double limit)
+{
+    // helper function to check if t is within limit percentage relative
+    //  to the center of the mesh
+    CAROM_VERIFY(t.Size() == bb_min.Size() && bb_min.Size() == bb_max.Size());
+    CAROM_VERIFY(limit >= 0.0 && limit <= 1.0);
+    for (int i = 0; i < t.Size(); i++)
+    {
+        double domain_limit = limit * (bb_max[i] - bb_min[i]);
+        double mesh_center = 0.5 * (bb_max[i] + bb_min[i]);
+
+        // check that t is within the limit relative to the center of the mesh
+        if (std::abs((t(i) - mesh_center)) - (0.5 * domain_limit) > 1.0e-14)
```

Comment:
It looks like the condition being checked here is
`if (std::abs(t(i) - mesh_center) < 0.5 * domain_limit)`
The current version is equivalent, but less clear.